README.md

```diff
@@ -7,7 +7,7 @@ ESLint configuration & ruleset for Javascript (ES6) + React.
 In order to add eslint-config-fiverr to a repository, use npm to add it to the development dependencies.
 
 ```
-npm i --save-dev eslint-plugin-react @fiverr/eslint-config-fiverr
+npm i -D @fiverr/eslint-config-fiverr [eslint-plugin-react]
```

Comment:
This will probably fail some people trying to copy and paste this line.
How about:

```
npm i -D @fiverr/eslint-config-fiverr
```

If you're extending the `react` module, make sure to install `eslint-plugin-react` as well:
```
npm i -D eslint-plugin-react
```

Or, a better option would be to just install it as a `dependency` of this package.